### PR TITLE
global: removal of Invenio-Records-UI

### PR DESCRIPTION
**Pull request by jirikuncar**
* Removes `record_file_factory` extenstion property. (addresses #134)

* Moves `file_download_ui` utility function to Invenio-Records-Files.

Signed-off-by: Jiri Kuncar <jiri.kuncar@cern.ch>

### CODE REVIEW

File: invenio_files_rest/ext.py [hunk 1]
```diff
@@ -45,21 +45,6 @@ def __init__(self, app):
         self.app = app
 
     @cached_property
-    def record_file_factory(self):
-        """Load default storage factory."""
-        r = load_or_import_from_config(
-            'FILES_REST_RECORD_FILE_FACTORY', app=self.app)
```
Comment (lnielsen) [inline invenio_files_rest/ext.py]: ``FILES_REST_RECORD_FILE_FACTORY`` should be removed from ``config.py`` as well + perhaps a comment in the commit that it's removed.

Comment (jirikuncar) [inline invenio_files_rest/ext.py]: see https://github.com/inveniosoftware/invenio-files-rest/pull/135/files#diff-eab9e55bede160cd708a25e6f768b6e4L94

**Comment by lnielsen** [comment]
Will you make a PR to Invenio-Records-Files or should I do it?

**Comment by jirikuncar** [comment]
@lnielsen do you mean inveniosoftware/invenio-records-files#38 ?

**Comment by jirikuncar** [comment]
FYI be0ceed + release is needed in order to unblock inveniosoftware/invenio-records-files#38

**Comment by lnielsen** [comment]
LGTM

**Pull request merged**